templates/footer.html

```diff
@@ -48,7 +48,7 @@ <h2 class="font-heading-md text-medium margin-top-4 tablet:margin-top-0 margin-b
         </h2>
         <ul class="usa-list usa-list--unstyled font-sans-sm list-item-spacing-2">
           {% for item in navigation.contact %}
-            <li><a href="{{ item.href | url }}">{{ item.text }}</a></li>
+            <li><a href="{{ item.href | url }}" {% if item.class %}class="{{ item.class }}"{% endif %}>{{ item.text }}</a></li>
```

Comment:
This content, template, and data are all the same shape as the social_media partial. I'm going to see if we can DRY this out.